### DISCUSSION TITLE
Wiki says that the multi-tool is designed for repair, NOT for detaching. However, the multi-tool detach detonators from walls. Was this how it was intended?

### DISCUSSION THREAD

**Discussion by pavel4win** [FakeFishGames/Barotrauma#15087] [Ideas and suggestions]
Wiki says that the multi-tool is designed for repair, NOT for detaching.
However, the multi-tool detach detonators from walls.
When I picked up a screwdriver and a wrench, this combination of tools in my hands also detach the detonator.
Was this how it was intended?

**Comment by SomeRandomNoobKekeke** [comment]
https://discord.com/channels/103209684680323072/426913708917129217

**Reply by pavel4win** [reply]
Sorry discord is blocked in Russia.
I can use vpn to use Discord  but I don't need it.
Link to which channel did you send me?

**Reply by SomeRandomNoobKekeke** [reply]
@pavel4win https://github.com/Flowseal/zapret-discord-youtube
![BaroDev (ultra wide)](https://github.com/user-attachments/assets/d5ccd645-1019-4b6b-942f-cef02d27e045)
 
> Link to which channel did you send me?

To wiki editing, i mean, it's not the game trying to match the wiki, most likely wiki maintainers just didn't update this info

**Reply by pavel4win** [reply]
I understand. I just tried to draw the developers' attention to a mechanic that is not obvious.
For me, the screwdriver + wrench combination as well as the multi-tool should not detach a detonator. For me, this was unexpected because it is impossible to run wires to the detonator with a multi-tool.

**Comment by JeroonBosch** [comment]
In the upcoming update, the multi-tool no longer detaches detonators from walls. Closing this
